src/OpenTelemetry.Contrib.Extensions/README.md

```diff
@@ -0,0 +1,13 @@
+# OpenTelemetry .NET SDK Extensions
+
+[![nuget](https://img.shields.io/nuget/v/OpenTelemetry.Contrib.Extensions.svg)](https://www.nuget.org/packages/OpenTelemetry.Contrib.Extensions/)
+
+Contains useful extensions to the OpenTelemetry .NET SDK that are not part of
```

Comment:
Maybe we want to leave some flexibility here, e.g. call it `OpenTelemetry.Contrib.Future` and describe it as "a collection of features that we might add to the OpenTelemetry .NET SDK in the future"?

Reply:
How about `OpenTelemetry.Contrib.Preview`? For some reason that feels better to me than "future" maybe because that's what .NET uses?

Reply:
Sounds good to me. I mentioned "future" only because of my previous experience with Python (e.g. https://docs.python.org/3/library/__future__.html), feel free to pick a better name that fits .NET.

Reply:
Updated